bigip/resource_bigip_ltm_node.go

```diff
@@ -235,8 +233,7 @@ func resourceBigipLtmNodeUpdate(d *schema.ResourceData, meta interface{}) error
 
```

Comment:
You are missing the interval attribute in the Update function when it's an FQDN Node.